pyanaconda/modules/payloads/source/utils.py

```diff
@@ -27,7 +27,20 @@
 log = get_module_logger(__name__)
 
 __all__ = ["is_valid_install_disk", "find_and_mount_device", "find_and_mount_iso_image",
-           "MountPointGenerator"]
+           "has_network_protocol", "MountPointGenerator"]
+
+
+def has_network_protocol(url):
+    """Does the given URL have a network protocol?
+
+    :param url: a string with URL
+    :return: True or False
+    """
+    if not url:
+        return False
+
+    network_protocols = ["http:", "ftp:", "nfs:", "nfsiso:"]
```

Comment:
https is missing

Reply:
See #2321...

Reply:
Fixed. I have used the commit from https://github.com/rhinstaller/anaconda/pull/2321.

Reply:
Thank you!